src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/bulkscan/transformation/in/ExceptionRecord.java

```diff
@@ -1,28 +1,29 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.in;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
 import lombok.Getter;
-import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.validation.in.OcrDataField;
 
 import java.util.List;
 
 @Getter
+@Builder
 public class ExceptionRecord {
```

Comment:
->lib

Reply:
Moved into the BSP Commons Lib as requested